config/application.rb

```diff
@@ -15,8 +16,8 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     # Custom directories with classes and modules you want to be autoloadable.
-    config.autoload_paths += %w(/lib/extras /lib/seed /lib/job_helpers
-                                /lib/).map { |s| "#{config.root}#{s}" }
+    config.autoload_paths += %w[/lib/extras /lib/seed /lib/job_helpers
```

Comment:
yeah, on vesta i ended up changing the rubocop config when I updated -- I can do the same here if you prefer